### PR TITLE
Add `ci-failure` label to upstream report

### PR DESCRIPTION
**Pull request by jrbourbeau**
This should make sure our `ci-failure` label is automatically applied when an upstream test failure issue is opened

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -432,19 +432,14 @@ jobs:
...
           github-token: ${{ secrets.GITHUB_TOKEN }}
           script: |
-            const title = "⚠️ CI failed ⚠️"
             const workflow_url = `https://github.com/${process.env.GITHUB_REPOSITORY}/actions/runs/${process.env.GITHUB_RUN_ID}`
             const issue_body = `[Workflow Run URL](${workflow_url})`
-            const variables = {
-                owner: context.repo.owner,
-                name: context.repo.repo,
-                creator: "github-actions[bot]"
-            }
```
Comment (jrbourbeau) [inline .github/workflows/tests.yml:442]: This is an unrelated simplification. It looks like we can just inline these values directly below.

**Review by ian-r-rose — APPROVED** [review]
Thanks @jrbourbeau
